.github/workflows/stale.yml

```
@@ -0,0 +1,29 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          # Aggressively close issues that have been explicitly labeled `age-out`
+          any-of-labels: age-out
+          stale-issue-message: 'This issue is stale because it has been open for 7 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
```

```suggestion
          stale-issue-message: 'This issue is stale because it has been open for 7 days with no activity. Remove age-out label or comment or this will be closed in 5 days.'
```

Or maybe just rename the label from `age-out` to `stale`? That seems to be a more common practice.

---

Oh nvm I misunderstood how this worked.

---

Okay I think the fix should be
```suggestion
          stale-issue-message: 'This issue is stale because it has been open for 7 days with no activity. Remove stale label or comment or this will be closed in 1 day.'
```